src/controllers/taskCommand.ts

```diff
@@ -0,0 +1,42 @@
+import { env } from "../typeDefinitions/default.types";
+import { TasksResponseType } from "../typeDefinitions/task.types";
+import { formatTask, generateTaskResponseMessage } from "../utils/formatTask";
+import { discordTextResponse } from "../utils/discordResponse";
+import { fetchTasks } from "../utils/fetchTasks";
+import { getNickName } from "../utils/getNickName";
+import {
+  INVALID_NICKNAME_ERROR,
+  NO_TASKS_FOUND,
+  TASKS_FETCH_FAILED,
+} from "../constants/responses";
+
+export async function taskCommand(userId: string, env: env) {
+  try {
+    const nickName = await getNickName(userId, env);
```

Comment:
is it possible to also declare the return type of this function?

Reply:
not needed

Reply:
why??

Reply:
When assigning a value to a variable, there is no need to specify its data type.